src/main/java/mate/academy/springboot/swagger/dto/RequestDto.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.springboot.swagger.dto;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Positive;
+import lombok.Data;
+
+@Data
+public class RequestDto {
```

Comment:
Not a big deal, but if we were to add more models to this application, this naming would be a little bit confusing. It would be better to call this class ProductRequestDto